Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_9.md

```diff
@@ -0,0 +1,23 @@
+
+#### Playbooks
+##### CrowdStrike Falcon - Get Detections by Incident
+Updated playbook description.
+##### CrowdStrike Falcon Malware - Verify Containment Actions
+Updated playbook description.
+##### CrowdStrike Falcon Malware - Incident Enrichment
+Updated playbook description.
+##### CrowdStrike Falcon - True Positive Incident Handling
+Updated playbook description.
+##### CrowdStrike Falcon - SIEM ingestion Get Incident Data
+Updated playbook description.
+##### CrowdStrike Falcon - False Positive Incident Handling
+Updated playbook description.
+##### CrowdStrike Falcon Malware - Investigation and Response
+- Updated playbook description.
+- Added additional filtering for file detonation logic.
```

Comment:
let's change it to "Fixed file detonation logic" or "Added validation for file detonation logic"

Reply:
Done.